### PR TITLE
Add --watch to impersonate

### PR DESCRIPTION
**Pull request by wzshiming**
Add --watch to impersonate subcommand

### CODE REVIEW

File: install/ctr-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        prometheus.io/scrape: "true"
+        prometheus.io/port: "8765"
+      labels:
+        app: adsc-load
+    spec:
+      serviceAccountName: pilot-load
+      terminationGracePeriodSeconds: 2000
+      containers:
+      - name: pilot-load
+        image: gcr.io/howardjohn-istio/pilot-load
```
Comment (wzshiming) [inline install/ctr-deployment.yaml:40]: I have tested with https://kwok.sigs.k8s.io/docs/examples/istio/.

**Review by howardjohn — COMMENTED** [review]
This is already done in https://github.com/howardjohn/pilot-load/blob/80c1a4066e368b0e5caf557aacf967b74ec0ca0a/cmd/impersonate.go#L22 I think?

**Comment by wzshiming** [comment]
It doesn't look the same, `impersonate` doesn't seem to dynamically connect/disconnect as the fake pod changes, it just seems to be connected in a real pod. let me know if I've misunderstood.

In other words, my intention is that there is an xds connection generator, which is controlled through a simulated pod. as this https://github.com/istio/istio.io/pull/13588

**Comment by howardjohn** [comment]
it could be updated to dynamically change if that is the only issue. We
don't need a bunch of commands that do 99% the same thing

**Comment by wzshiming** [comment]
Okay, I'll try it.

**Review by howardjohn — APPROVED** [review]

**Pull request merged**